example/mock_secret_backend/src/mock_secret_backend.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python
+
+import sys
+import json
+
+def main():
```

Comment:
Let's add the link to the agent doc. So that we make it clear that this script is just the python version of the example.